Xamarin.Forms.Core/Properties/AssemblyInfo.cs

```diff
@@ -61,6 +61,7 @@
 [assembly: StyleProperty("text-align", typeof(ITextAlignmentElement), nameof(TextAlignmentElement.HorizontalTextAlignmentProperty), Inherited = true)]
 [assembly: StyleProperty("visibility", typeof(VisualElement), nameof(VisualElement.IsVisibleProperty), Inherited = true)]
 [assembly: StyleProperty("width", typeof(VisualElement), nameof(VisualElement.WidthRequestProperty))]
+[assembly: StyleProperty("line-height", typeof(Label), nameof(Label.LineHeightProperty))]
```

Comment:
`line-height` is inherited in CSS, set `Inherited = true`

Reply:
that won't set the line-height on Span. both Span and Label have to implement a common interface (look at `IFontElement`)